enterprise/views_error_codes.txt

```diff
@@ -10,6 +10,7 @@ ENTGDS004 - Course catalog api configuration error
 ENTGDS005 - Required request values missing for action to be carried out
 ENTGDS006 - The course or program with a given id does not exist
 ENTGDS007 - There is no consent record, or consent is not required
```

Comment:
@christopappas Any idea why we have duplicate messages with different codes? 
ENTGDS000 and ENTGDS006 are same
ENTGDS002 and ENTGDS007 are same
ENTGDS003 and ENTGDS005 are same

Reply:
The codes themselves are more for indicating the flow in which the error occurred, and not the message that was returned to the user. It happens that some of the messages are the same, because we do similar checks in a few places, but we needed unique codes cause they're in different code paths